sdk/ai/azure-ai-projects/samples/red_team/sample_red_team.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
```

Comment:
Please run the "black" tool at the project root (\sdk\ai\azure-ai-projects) to fix code alignment issues:
black --config ../../../eng/black-pyproject.toml .